index.js

```diff
@@ -210,10 +210,13 @@ var lastResorts = [
   ['BEER', 'Beerware'],
   ['BOOST', 'BSL-1.0'],
   ['BSD', 'BSD-2-Clause'],
+  ['CDDL', 'CDDL-1.1'],
```

Comment:
HI @zommerfelds :)
The motivation is that "CDDL" was not identified. I chose 1.1 just because it's more recent, just like "GPL" is inferred to be "GPL-3.0" in this last resort section. Not sure it's the right choice. 
Of course, a proper string like "CDDL-1.0" or "CDDL-1.1" is already recognised correctly, prior to this change.